Production/govuk_ios/SupportingFiles/Settings.bundle/ar.lproj/Root.strings

```diff
@@ -0,0 +1 @@
+"Acknowledgements" = "مخزن";
```

Comment:
Maybe I need to PR into the lib to add Welsh.
We had said no Welsh support for Beta, but I gues if we have all the others, we should probably put Welsh in as well 👍🏻

Reply:
This is an interesting one. Given iOS doesn't provide language localisation for Welsh and I suspect most of these licences are only available in English due to the legal nature of them, I'm kind of of the opinion that providing a few welsh words within an otherwise all english context (because Settings app is an Apple iOS thing) might be a bit peculiar? Something to discuss with the content designers.

Reply:
@robinphillips Ah, it's not the Licenses that will need to be converted to and stored/displayed in Welsh, but everything else in the app! All the content, labels, help, navigation and so on.

Reply:
Example: 

- https://www.gov.uk/sign-in-universal-credit (English)
- https://www.gov.uk/mewngofnodi-ich-cyfrif-credyd-cynhwysol (Welsh)

Reply:
Yes for sure. We went through that on ID Check :) 

But this PR is adding stuff to the iOS Settings app within iOS where nothing else will have welsh. So to try to localise 'licences' or 'acknowledgements' into welsh the context of the settings app where everything else would be in English would probably not be right.

There are other system alerts where the choice was made to not localise into Welsh for similar reasons because some content was controlled by iOS and couldn't be localised to Welsh so would have created a jarring experience.

Reply:
So, there's no reason to convert into German, Arabic etc and those files can be removed from the PR?

Reply:
iOS will likely have language localisation for those built in. I'm not 100% if it has all of them, but probably. It definitely does not have Welsh though.

The result of adding a Welsh headings here would be that the whole of the settings app and the licence would be in English with just a heading that said 'licence' in Welsh.

You can experiment by putting an iPhone (or the simulator) in Welsh and then navigating around. Everything will remain displayed in English until you open an app that has been localised for Welsh.

Reply:
I would probably take the view that we should do localisation as part of a product decision. And then be strategic about it. We can then add translation that we know is right and in the right places as per the content rather than us deciding to add it for some languages because we can.